pom.xml

```diff
@@ -30,7 +30,7 @@
   </scm>
 
   <properties>
-    <changelist>999999-SNAPSHOT</changelist>
+    <changelist>1781.va_153cda_09d1b_dock</changelist>
```

Comment:
Revert please. Note that any PR which passes tests will automatically deploy a binary to the Incrementals repository.